File: FizzBuzz Algorithm using C++/Fizz_Buzz.cpp
#include<iostream>
using namespace std;
int main(){
    for(int i = 1; i<=20; i++){
        if(i % 3 == 0 && i % 5 == 0){
            cout<<"FizzBuzz"<<endl;
        }
        else if(i % 3 == 0){
            cout<<"Fizz"<<endl;
        }
        else if(i % 5 == 0){
            cout<<"Buzz"<<endl;
        }
        else{
            cout<<i<<endl;
        }
    }
    return 0;
}

File: Python-programming-1/Fizz-Buzz.py
#Famous problem on CodeChef and LeetCode is Fizz Buzz problem
'''
The concept of this problem is that
if the number entered by user is divisible by 3 and not by 5, then we print Fizz on Console
if the number entered by user is divisible by 5 and not by 3, then we print Buzz on Console
if the number entered by user is divisible by 3 and 5 both, then we print Fizz Buzz on Console
'''

num = int(input())
if(num%3==0 and num%5==0):
    print("Fizz Buzz")
elif(num%3==0 and num%5!=0):
    print("Fizz")
elif(num%3!=0 and num%5==0):
    print("Buzz")
else:
    print("Entered number is neither divisible by 3 nor 5")